.github/workflows/release.yml

```diff
@@ -22,3 +22,5 @@ jobs:
         run: ./push-package.sh
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          confirm: "y"
+          SKIP_TAG: "y"
```

Comment:
Why don't we want to create a git tag for every npm release? Wouldn't that be better?